Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,55 @@
+﻿namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
+
+        public Account(string personAndBalance)
+        {
+            string[] _accountArray = ArrayOps.ConvertToAccountArray(personAndBalance);
+            _name = AccountOps.GetNameForPerson(_accountArray);
+            _balance = AccountOps.GetBalanceForPerson(_accountArray);
+        }
+
+        public string GetName()
+        {
+            return _name;
+        }
+
+        public decimal[] GetBalance()
+        {
+            return _balance;
+        }
+
+        public decimal GetHighestBalance()
+        {
+            var highestBalance = ArrayOps.FindArrayMax(_balance);
+            return highestBalance;
+        }
+
+        public decimal GetLoss()
```

Comment:
It's the last loss, isn't it? Maybe it would make sense to give it an index and return loss of a certain historical balance?

Reply:
I refactored it in [d1518fc](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/192/commits/d1518fc7884307a9ddc2f6074fa792682af6878d). Thank you for the sugestion.